internal/acctest/acctest.go

```diff
@@ -266,6 +277,51 @@ func AddIdHcl(idKey, id string) string {
 	return ""
 }
 
+func ReturnStateMismatchError(stateKey, stateValue, configValue string) error {
+	return fmt.Errorf("state value for %s does not match config value. State: %s, Found: %s", stateKey, stateValue, configValue)
+}
+
+func VerifyStateAttributeValue(stateAttributes map[string]string, stateKeyValue string, configValue interface{}) error {
```

Comment:
Do we need this custom method? I thought the testing framework had its own built-in methods to verify state that we use in a few tests.